pkg/imagestore/imagestore.go

```diff
@@ -84,16 +86,65 @@ const (
 	ImageTypeMinimal = "minimal-iso"
 )
 
-func NewImageStore(ed isoeditor.Editor, dataDir, imageServiceBaseURL string, insecureSkipVerify bool, versions []map[string]string) (ImageStore, error) {
+func getAdditionalTLSCertPaths(additionalCAPath string) ([]string, error) {
+	if strings.Trim(additionalCAPath, " ") == "" {
```

Comment:
` ` is a valid file name, no need to do trimming here. If someone is putting spaces in their environment variables, I'd like to tell them about it with an explicit error of "file not found", not pretend that everything is fine